### PR TITLE
Updating token deprecation date

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Adding date for old token deprecation

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/dbt-cloud-apis/user-tokens.md [hunk 1]
```diff
@@ -8,7 +8,10 @@ pagination_next: "docs/dbt-cloud-apis/service-tokens"
 
 :::note Announcement
 
-The [user API tokens](#user-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). We recommend rotating your existing user tokens with PATs. There are no deprecation plans for user API tokens at this time; we will give ample notice when that timeline has been determined. 
+_User tokens will be deprecated on September 18th, 2024._
+
+The [user API tokens](#user-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). If you do not rotate your existing user tokens with PATs by September 18th, the services using the tokens will encounter errors. 
+
```
Comment (nataliefiann) [inline website/docs/docs/dbt-cloud-apis/user-tokens.md:14]: Hiya @matthewshaver 

Great PR and no real comments or suggestions. I did notice the first link for API Scoped User Tokens didn't direct me to the relevant page / section. I've added the link for ref (/docs/dbt-cloud-apis/user-tokens#user-api-tokens).

Kind Regards
Natalie

File: website/docs/docs/dbt-cloud-apis/user-tokens.md [hunk 2]
```diff
@@ -8,7 +8,10 @@ pagination_next: "docs/dbt-cloud-apis/service-tokens"
 
 :::note Announcement
 
-The [user API tokens](#user-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). We recommend rotating your existing user tokens with PATs. There are no deprecation plans for user API tokens at this time; we will give ample notice when that timeline has been determined. 
+_User tokens will be deprecated on September 18th, 2024._
+
+The [user API tokens](#user-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). If you do not rotate your existing user tokens with PATs by September 18th, the services using the tokens will encounter errors.
```
Comment (nataliefiann) [inline website/docs/docs/dbt-cloud-apis/user-tokens.md]:
```suggestion
The [user API tokens](/docs/dbt-cloud-apis/user-tokens#user-api-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). If you do not rotate your existing user tokens with PATs by September 18th, the services using the tokens will encounter errors. 
```

**Review by nataliefiann — APPROVED** [review]
Hiya @matthewshaver 

Thanks for flagging this. I've made one suggestion but otherwise the doc is perfect. I'll approve this for you in the meantime.

Kind Regards
Natalie

File: website/docs/docs/dbt-cloud-apis/user-tokens.md (hunk 2 above)
Comment (matthewshaver) [reply website/docs/docs/dbt-cloud-apis/user-tokens.md]: Good catch! Thank you!

**Pull request merged**